frameworks/kafka/universe/config.json

```diff
@@ -698,13 +765,19 @@
           "title": "log.cleaner.io.max.bytes.per.second",
           "description": "The log cleaner will be throttled so that the sum of its read and write i/o will be less than this value on average",
           "type": "number",
-          "default": 1.7976931348623157e+308
```

Comment:
The docs I saw still referenced the old value.

Reply:
Heh, this is the result of me parsing it to a value.

The docs have the old value.

Reply:
Good old floating-point arithmetic! I still remember the days where that was what put food on the table.